src/charm.py

```diff
@@ -227,6 +245,64 @@ def _on_database_storage_detaching(self, _) -> None:
         # Inform other hooks of current status
         self._peers.data[self.unit]["unit-status"] = "removing"
 
+    def _on_update_status(self, _) -> None:
+        """Handle update status.
+
+        Takes care of workload health checks and.
```

Comment:
Comment here has a trailing `and` - did you mean to add something else to the sentence?

Reply:
fixed